src/nls/messages_zh-Hans.json

```diff
@@ -1,256 +1,256 @@
+    "dashboard.createPipelineResource.revision": "pipeline-resource-revision",
+    "dashboard.createPipelineResource.revisionError": "修改历史是必须的",
+    "dashboard.createPipelineResource.revisionLabel": "修改历史",
+    "dashboard.createPipelineResource.title": "创建 PipelineResource",
+    "dashboard.createPipelineResource.type": "类型",
+    "dashboard.createPipelineResource.url": "pipeline-resource-url",
+    "dashboard.createPipelineResource.urlError": "URL 是必须的",
+    "dashboard.createPipelineResource.urlLabel": "URL",
+    "dashboard.createPipelineRun.createError": "创建 PipelineRun 时失败",
+    "dashboard.createPipelineRun.errorLoading": "检索 Pipeline 信息时失败",
+    "dashboard.createPipelineRun.invalidPipeline": "Pipeline 不能为空",
+    "dashboard.createPipelineRun.serviceAccountHelperText": "确保所选的 ServiceAccount（如果没有选择，则为默认值）具有创建 PipelineRun 和与 PipelineRun 交互的任何其他权限。",
+    "dashboard.createPipelineRun.timeoutHelperText": "PipelineRun 的超时（分钟）",
+    "dashboard.createPipelineRun.title": "创建 PipelineRun",
+    "dashboard.createRun.invalidNamespace": "Namespace 不能为空",
+    "dashboard.createRun.invalidParams": "Params 不能为空",
+    "dashboard.createRun.invalidPipelineResources": "PipelineResources 不能为空",
+    "dashboard.createRun.invalidTimeout": "Timeout 必须是一个小于 525600 的有效数字",
+    "dashboard.createRun.label.invalidText": "Label 必须遵循 {0} Kubernetes 标签语法 {1}",
+    "dashboard.createRun.labels.legendText": "标签",
+    "dashboard.createRun.nodeSelector.legendText": "节点选择器",
+    "dashboard.createRun.optional.legendText": "可选值",
+    "dashboard.createRun.timeoutLabel": "超时",
+    "dashboard.createRun.validationError": "请修正有错误的字段，然后重新提交。",
+    "dashboard.createSecret.accessTo": "访问",
+    "dashboard.createSecret.accessToDocker": "Docker Registry",
+    "dashboard.createSecret.accessToGit": "Git 服务器",
+    "dashboard.createSecret.addAnnotations": "添加注释",
+    "dashboard.createSecret.duplicateErrorMessage": "Secret {name} 在 Namespace {namespace} 中已存在",
+    "dashboard.createSecret.finishButton": "完成",
+    "dashboard.createSecret.heading": "创建新的 Secret",
+    "dashboard.createSecret.patchButton": "Patch",
+    "dashboard.createSecret.serverURLInvalid": "服务器 URL 是必须的",
+    "dashboard.createSecret.serviceAccountSelectorSubHeading": "选择要将该 Secret 添加到的 ServiceAccount（可选）",
+    "dashboard.createSecret.successMessage": "成功创建 {name} ",
+    "dashboard.createSecret.title": "创建 Secret",
+    "dashboard.createTaskRun.createError": "创建 TaskRun 时失败",
+    "dashboard.createTaskRun.errorLoading": "检索 Task 信息时失败",
+    "dashboard.createTaskRun.invalidTask": "Task 不能为空",
+    "dashboard.createTaskRun.serviceAccountHelperText": "确保所选的（若未选择则为默认的）ServiceAccount 具有创建 TaskRun 和与 TaskRun 交互的任何其他权限。",
+    "dashboard.createTaskRun.timeoutHelperText": "TaskRun 的超时（分钟）",
+    "dashboard.createTaskRun.title": "创建 TaskRun",
+    "dashboard.deletePipelineResource.body": "您确定要删除 PipelineResource {0} 吗",
+    "dashboard.deletePipelineRun.body": "您确定要删除 PipelineRun {0} 吗",
```

Comment:
![image](https://user-images.githubusercontent.com/2829095/106193779-e0908d80-61a5-11eb-94ed-f4e225b92cb4.png)

The placeholder should be `name` instead of `0`:

```suggestion
    "dashboard.deletePipelineRun.body": "您确定要删除 PipelineRun {name} 吗",
```

Reply:
FIxed.